lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,133 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+<div class="calc-main">
+    <a title="${_('Open Calculator')}" href="#" role="button" aria-controls="calculator_wrapper" aria-expanded="false" class="calc">${_("Open Calculator")}</a>
```

Comment:
My understanding is that a ``<button>`` element should be used in a ``<form>``. See - http://css-tricks.com/use-button-element/. I tend to agree with Chris Coyier with all of the scenarios brought up there.